src/components/common/Container/Container.tsx

```diff
@@ -17,16 +17,20 @@ export const SubContainer = styled.div<{ justifyValue: string }>`
   flex-direction: row;
   align-items: center;
   height: 500px;
+  min-width: 1100px;
   justify-content: ${(props) => props.justifyValue};
 `;
 
-const Container = (props: any) => {
+type WrapperProps = {
+  children: JSX.Element;
+  justifyValue: string;
+};
```

Comment:
```suggestion
interface ContainerProps = {
  children: JSX.Element;
  justifyValue: string;
};
```

해당 파일 이름과 똑같이 유지하는 편이 좋다고 생각합니다. 상속받아서 사용하는 경우가 아니라면 그냥 Props만 적어도 괜찮다고 생각하기도 하구요